front/lib/api/assistant/conversation.ts

```diff
@@ -1027,7 +1027,9 @@ export async function* postUserMessage(
   async function logIfUserUnknown() {
     try {
       if (!user && context.email) {
-        const macthingUser = await UserResource.fetchByEmail(context.email);
+        const macthingUser = await UserResource.fetchByEmail(
+          context.email.toLowerCase()
```

Comment:
I guess we can remove it from here, as it's done in `fetchByEmail`?

Reply:
indeed